Gruntfile.js

```diff
@@ -807,6 +833,8 @@ module.exports = function (grunt) {
     grunt.registerTask('build-dev-mv3', [
         'clean:intermediates',
         'exec:generate-scss-typings',
+        'build-package-validator',
```

Comment:
I think this should also be in _somewhere_ in build:all as well, no?

Reply:
As far as I can tell, it does not currently run there?

Reply:
I figured it would be redundant to add build-package-validator explicitly to build:all since it's already implicitly included through build-dev-mv3, do you think it's worthwhile to add is explicitly so it runs multiple times?